cloud-sql/postgres/activerecord/app.flexible.yaml

```diff
@@ -14,15 +14,15 @@
 
 entrypoint: bundle exec rackup --port $PORT
 env: flex
-runtime: ruby
+runtime: custom
```

Comment:
Please revert to `runtime: ruby`. If you want to use a different version of Ruby, provide a `.ruby-version` file. For example, if you want to use a release of Ruby 2.7 to match the runtime you're using for App Engine Standard, create, in the same location as the app.yaml files, a file called `.ruby-version` with the contents:

```
2.7.5
```

Reply:
Done.